### ISSUE TITLE
Resume after interruption

### ISSUE THREAD

**Issue by rfsaldanha** [DavisVaughan/furrr#249]
This is not an issue but a suggestion.

Some computations even in parallel may take a long time and it will be useful to have a way to resume the computation after a user interruption of the process.

**Comment by DavisVaughan** [comment]
Unfortunately I think that is out of scope for furrr. I appreciate the suggestion though
